provider/resource_keycloak_openid_client_optional_scopes_test.go

```diff
@@ -429,6 +440,37 @@ resource "keycloak_openid_client_optional_scopes" "optional_scopes" {
 	]
 }
 	`, realm, client, clientScope)
+	} else {
+		return fmt.Sprintf(`
+resource "keycloak_realm" "realm" {
+	realm = "%s"
+}
+
+resource "keycloak_openid_client" "client" {
+	client_id   = "%s"
+	realm_id    = "${keycloak_realm.realm.id}"
+	access_type = "PUBLIC"
+}
+
+resource "keycloak_openid_client_scope" "client_scope" {
+	name        = "%s"
+	realm_id    = "${keycloak_realm.realm.id}"
+
+	description = "test description"
+}
+
+resource "keycloak_openid_client_optional_scopes" "optional_scopes" {
+	realm_id       = "${keycloak_realm.realm.id}"
+	client_id      = "${keycloak_openid_client.client.id}"
+	optional_scopes = [
+		"address",
+		"phone",
+		"offline_access",
+		"${keycloak_openid_client_scope.client_scope.name}"
+	]
+}
+	`, realm, client, clientScope)
+	}
```

Comment:
I am aware we could work with smaller yaml string snippets and concat them. But I went for clarity and use the whole yml snippet as it would be used in that version of keycloak